### PR TITLE
UI Improvements Part 1: Home page

### PR DESCRIPTION
**Pull request by marshall2112**
⚠️ Gamma already approved the changes in discord FYI https://discord.com/channels/847178511604252741/949614784079343616/1027796224813842472

# Description

What does this PR solve?

The UI feedback for the home page.

Screen Shot 2022-10-07 at 12 03 06 AM

Screen Shot 2022-10-03 at 4 28 21 PM

Fixes # (issue)

# Checklist
- [ ] Code follows the style guide
- [ ] I have performed a self-review of my own code
- [ ] New and existing tests pass locally
- [ ] This PR is targeting the correct branch

### CODE REVIEW

File: apps/dapp/src/components/Metrics/Metrics.tsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ const Metrics = ({ treasuryMetrics, isHome, alignCenter }: MetricsProps) => {
           cryptoName={'vAPY'}
           value={`${formatNumber(dynamicApy)}%`}
           imageData={{
```
Comment (jebsley3) [inline apps/dapp/src/components/Metrics/Metrics.tsx:43]: See other comment about making this an optional prop

File: apps/dapp/src/components/Footer/Footer.tsx [hunk 2]
```diff
@@ -20,80 +20,34 @@ export const Footer = () => {
...
-            reserved.
-          </strong>
+          <CopyrightStyled>&copy; {new Date().getFullYear()} TempleDAO. All rights reserved.</CopyrightStyled>
         </div>
         <FooterGrid>
           <Column>
             <h4>Community</h4>
             <ul>
               <li>
                 {/* TODO: add no refere for SEO  */}
```
Comment (jebsley3) [inline apps/dapp/src/components/Footer/Footer.tsx:30]: Unrelated to your changes but do you know what this comment is about?

**Review by jebsley3 — APPROVED** [review]
Looks much nicer
Some comments but totally not blocking

File: apps/dapp/src/components/Footer/Footer.tsx (hunk 2 above)
Comment (marshall2112) [reply apps/dapp/src/components/Footer/Footer.tsx:30]: I am not sure, no sorry.

File: apps/dapp/src/components/Apy/Apy.tsx [hunk 3]
```diff
@@ -15,31 +15,16 @@ interface ApyProps extends ApyStyledProps {
...
-      isHome={isHome}
-    >
-      <Image
-        src={imageUrl}
-        alt={alt}
-        width={getImageSize()}
-        height={getImageSize()}
-      />
+    <ApyStyled alignCenter={alignCenter} isHome={isHome}>
+      {imageUrl !== '' && <Image src={imageUrl} alt={alt} width={getImageSize()} height={getImageSize()} />}
```
[the remark this answers is not in the document]
Comment (marshall2112) [reply apps/dapp/src/components/Apy/Apy.tsx]: https://github.com/TempleDAO/temple/pull/576/commits/6af1e9ea10513b33c23dfc47285cb125663b5969

File: apps/dapp/src/components/Metrics/Metrics.tsx (hunk 1 above)
Comment (marshall2112) [reply apps/dapp/src/components/Metrics/Metrics.tsx]: https://github.com/TempleDAO/temple/pull/576/commits/6af1e9ea10513b33c23dfc47285cb125663b5969

**Pull request merged**